src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+const mainList = document.querySelector('ul');
+
+function toNum(str) {
+  return +str.slice(1).split(',').join('');
+}
+
+function sortList(someList) {
+  return [...someList].sort((a, b) =>
+    toNum(a.dataset.salary) - toNum(b.dataset.salary)).reverse();
+}
+
+function getEmployees(someList) {
+  const arr = [];
+
+  for (const li of someList) {
+    const obj = {
+      name: li.textContent.trim(),
+      position: li.dataset.position,
+      salary: li.dataset.salary,
+      age: li.dataset.age,
+    };
+
+    arr.push(obj);
+  }
+
+  return arr;
+}
+
+const sorted = getEmployees(sortList(list));
+
+mainList.innerHTML = `${sorted.map(({ name, position, salary, age }) => `
```

Comment:
```suggestion
mainList.innerHTML = `${sorted.map(({ name: employeeName, position, salary, age }) => `
```
for fix linter error